pkg/provider/aws/execution_unit_test.go

```diff
@@ -35,6 +35,11 @@ func Test_GenerateExecUnitResources(t *testing.T) {
 				Type:         string(kubernetes.ImageTransformation),
 				Key:          "image",
 			},
+			{
+				ExecUnitName: unit.ID,
+				Type:         string(kubernetes.TargetGroupTransformation),
+				Key:          "tgb",
```

Comment:
tgb? (what's the "b"?)

Reply:
target group binding is the term for the actual manifest, but it doesnt matter what the key value is so i just put tgb